src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -265,8 +267,21 @@ const IssueForm = (): JSX.Element | null => {
         setSubmitStatus(STATUSES.PENDING);
         await requestLimitsRefetch();
         await trigger(BTC_AMOUNT);
+
         const wrappedTokenAmount = BitcoinAmount.from.BTC(data[BTC_AMOUNT] || '0');
-        const result = await window.bridge.issue.request(wrappedTokenAmount);
+
+        const vaults = await window.bridge.vaults.getVaultsWithIssuableTokens();
+        const vaultId = getRandomVaultIdWithCapacity(Array.from(vaults || new Map()), wrappedTokenAmount);
```

Comment:
@tomjeatt 
I think we do not need this fallback `|| new Map()`. It looks redundant.

Reply:
Yes you're right :)